ddtrace/contrib/internal/asgi/middleware.py

```diff
@@ -236,9 +236,20 @@ async def wrapped_send(message):
                     response_headers = None
 
                 if span and message.get("type") == "http.response.start" and "status" in message:
+                    cookies = {}
+                    try:
+                        cookie_key, *split_cookie = response_headers.get("set-cookie", "").split("=")
+                        cookies[cookie_key] = "=".join(split_cookie)
```

Comment:
```suggestion
                        cookie_key, cookie_value = response_headers.get("set-cookie", "").split("=", maxsplit=1)
                        cookies[cookie_key] = cookie_value
```

Reply:
I forgot the maxsplit parameter, nice suggestion @emmettbutler !! :clap: